src/appleseed/renderer/modeling/material/material.cpp

```diff
@@ -94,6 +94,20 @@ bool Material::has_alpha_map() const
     return get_non_empty(m_params, "alpha_map") != 0;
 }
 
+bool Material::has_uniform_opacity() const
```

Comment:
I thought a lot about naming this correctly. How about these:
`is_uniformly_opaque()` or a more expressive one `has_uniform_alpha_map_value_of_one()` ?
I am open to suggestions here:)